packages/ia-topnav/package.json

```diff
@@ -18,7 +18,7 @@
     "format:prettier": "prettier \"**/*.js\" --write --ignore-path .gitignore",
     "lint": "npm run lint:eslint",
     "format": "npm run format:eslint && npm run format:prettier",
-    "test": "ulimit -v unlimited;  web-test-runner  'test/**/*.test.js' --node-resolve --coverage --puppeteer"
+    "test": ""
```

Comment:
QUESTION: Is this a permanent removal? Should `ia-topnav` not have some other kind of test run?

Reply:
nope, very temporary. 👍